<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Graphicus</class>
 <widget class="QWidget" name="Graphicus">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>951</width>
    <height>500</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>951</width>
    <height>500</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>951</width>
    <height>500</height>
   </size>
  </property>
  <property name="baseSize">
   <size>
    <width>951</width>
    <height>500</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QLineEdit" name="toArea">
   <property name="geometry">
    <rect>
     <x>254</x>
     <y>394</y>
     <width>215</width>
     <height>70</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>215</width>
     <height>70</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>215</width>
     <height>70</height>
    </size>
   </property>
   <property name="font">
    <font>
     <pointsize>22</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="maxLength">
    <number>255</number>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="placeholderText">
    <string>X до</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="fromArea">
   <property name="geometry">
    <rect>
     <x>29</x>
     <y>394</y>
     <width>215</width>
     <height>70</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>215</width>
     <height>70</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>215</width>
     <height>70</height>
    </size>
   </property>
   <property name="font">
    <font>
     <pointsize>22</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="maxLength">
    <number>255</number>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="placeholderText">
    <string>X от</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_make">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>481</x>
     <y>395</y>
     <width>440</width>
     <height>70</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>440</width>
     <height>70</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>440</width>
     <height>70</height>
    </size>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(107, 182, 255);</string>
   </property>
   <property name="text">
    <string>Построить график</string>
   </property>
  </widget>
  <widget class="QCustomPlot" name="gWin" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>951</width>
     <height>361</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>951</width>
     <height>361</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>951</width>
     <height>361</height>
    </size>
   </property>
   <widget class="QLabel" name="expLabel">
    <property name="geometry">
     <rect>
      <x>37</x>
      <y>30</y>
      <width>881</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 0);
background-color: rgb(255, 255, 255, 0.0);</string>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="expLabel_2">
    <property name="geometry">
     <rect>
      <x>857</x>
      <y>14</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 0);
background-color: rgb(255, 255, 255, 0.0);</string>
    </property>
    <property name="text">
     <string>Формула:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
